spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/Mqttv5PahoMessageDrivenChannelAdapter.java

```diff
@@ -467,7 +472,11 @@ private void subscribe() {
 			IMqttMessageListener[] listeners = IntStream.range(0, mqttSubscriptions.length)
 					.mapToObj(t -> listener)
 					.toArray(IMqttMessageListener[]::new);
-			this.mqttClient.subscribe(mqttSubscriptions, null, null, listeners, new MqttProperties())
+			MqttProperties subsProperties = new MqttProperties();
```

Comment:
DITTO

Reply:
Fixed.

Reply:
No need to say `Fixed` if you agree with me and going to incorporate that in the next commit for the PR.
I'll see those changes this or other way.
Otherwise this sounds like a spam in my email box 😄 

Only exclusion is a force-push where GitHub don't generate email notification for some reason.